server/src/main/java/com/epam/aidial/core/server/controller/AppSchemasController.java

```diff
@@ -48,7 +51,8 @@ public Future<?> handle() {
 
     private Future<?> handleGetMetaSchema() {
         try {
-            return context.respond(HttpStatus.OK, MetaSchemaHolder.getCustomApplicationMetaSchema());
+            return vertx.executeBlocking(MetaSchemaHolder::getCustomApplicationMetaSchema)
+                    .onSuccess(metaSchema -> context.respond(HttpStatus.OK, metaSchema));
```

Comment:
Move the code from catch into .onFailure((e) -> {}) after onSuccessMethod

Reply:
Do not want to because the target method used not only in that controller